modules/artifact-registry/README.md

```diff
@@ -210,28 +211,95 @@ module "registry-docker" {
 }
 # tftest modules=1 resources=1 inventory=cleanup-policies.yaml
 ```
+
+## IAM
+
+This module implements the same IAM interface than the other modules. 
+You can choose one (and only one) of the three options below:
+
+```hcl
+# Authoritative IAM bindings
+module "authoritative_iam" {
+  source     = "./fabric/modules/artifact-registry"
+  project_id = "myproject"
+  location   = "europe-west1"
+  name       = "myregistry"
+  format     = { docker = { standard = {} } }
+  iam = {
+    "roles/artifactregistry.admin" = ["group:cicd@example.com"]
+  }
+}
+# tftest modules=1 resources=1 inventory=iam.yaml
+```
+```hcl
+# Authoritative IAM bindings (with conditions)
+module "authoritative_iam_conditions" {
+  source     = "./fabric/modules/artifact-registry"
+  project_id = "myproject"
+  location   = "europe-west1"
+  name       = "myregistry"
+  format     = { docker = { standard = {} } }
+  iam_bindings = {
+    "ci-admin" = {
+      member = "group:cicd@example.com"
+      role   = "roles/artifactregistry.admin"
+      // condition = {
+      //   expression  = string
+      //   title       = string
+      //   description = optional(string)
+      // }
+    }
+  }
+}
+# tftest modules=1 resources=1 inventory=iam-bindings.yaml
+```
+```hcl
+# Additive IAM bindings
+module "additive_iam" {
+  source     = "./fabric/modules/artifact-registry"
+  project_id = "myproject"
+  location   = "europe-west1"
+  name       = "myregistry"
+  format     = { docker = { standard = {} } }
+  iam_bindings_additive = {
+    "ci-admin" = {
+      member = "group:cicd@example.com"
+      role   = "roles/artifactregistry.admin"
+      // condition = {
+      //   expression  = string
+      //   title       = string
+      //   description = optional(string)
+      // }
+    }
+  }
+}
+# tftest modules=1 resources=1 inventory=iam-bindings-additive.yaml
```

Comment:
I think you can remove the commented part and merge both examples in a single one.

Also, you're missing the inventory files.

Reply:
trick for inventories without you having to wade through the contributing guide:

- set up your virtualenv with the reqs in tests
- `pytest -s tests/examples -k modules/artifact-registry`
- copy/paste inventories

Reply:
Thanks 🙇